main.py

```diff
@@ -15,27 +14,100 @@ class Robot:
     UPPER_POSITION = 30
     LOWER_POSITION = 0
     WIGGLE_AMOUNT = 5
+    INACTIVITY_PERIOD_S = 5
 
     async def connect(self):
         opts = RobotClient.Options(
             refresh_interval=0,
             dial_options=DialOptions(credentials=secrets.creds)
         )
         self.robot = await RobotClient.at_address(secrets.address, opts)
-        self.servo = Servo.from_robot(self.robot, "servo")
+        self._servo = Servo.from_robot(self.robot, "servo")
+
+        self._mutex = asyncio.Lock()
+        self._count = 0  # Number of hands currently raised
+
+        # self._wiggler will become an asyncio.Task when the hand is raised. It
+        # will wiggle the hand when it has been raised for over
+        # INACTIVITY_PERIOD_S seconds. When the hand is lowered, we set the
+        # Event to tell the task to stop.
+        self._wiggler = None
+        self._shut_down_wiggler = asyncio.Event()
+
+    def _start_wiggler(self):
+        self._shut_down_wiggler.clear()
+        self._wiggler = asyncio.create_task(self._wiggle_on_inactivity())
```

Comment:
Now that this is a Task, I suspect it is cancellable, without needing a separate Event to shut it down. Look into that tomorrow.